features/support/publishing_cuke_helpers.rb

```diff
@@ -27,7 +27,7 @@ def make_post(text)
 
   def submit_publisher
     txt = find('#publisher #status_message_fake_text').value
-    find('#publisher .creation').click
+    find('#publisher .btn-primary').click
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.